core/modules/replica/tools/qserv-replica-messenger-test.cc

```diff
@@ -0,0 +1,154 @@
+#include <atomic>
+#include <iomanip>
+#include <iostream>
+#include <memory>
+#include <stdexcept>
+#include <string>
+
+#include "proto/replication.pb.h"
+#include "replica/Configuration.h"
+#include "replica/Controller.h"
+#include "replica/Messenger.h"
+#include "replica/MessengerConnector.h"
+#include "replica/ProtocolBuffer.h"
+#include "replica/ServiceProvider.h"
+#include "util/BlockPost.h"
+#include "util/CmdLineParser.h"
+
+namespace proto   = lsst::qserv::proto;
+namespace replica = lsst::qserv::replica;
+namespace util    = lsst::qserv::util;
```

Comment:
I think you could also get away with `using namespace lsst::qserv`, but ok.

Reply:
Good suggestion! Fixed.